docs/docs/user/migration-guide/v2-v3.md

```diff
@@ -81,3 +81,16 @@ The `--config` flag in the CLI now points to the directory that contains `config
     enabled: true
     url: "http://my-custom-sr.url:8081"
 ```
+
+## [Define custom components module & pipeline base dir globally](https://github.com/bakdata/kpops/pull/387)
+
+The options for a custom `components_module` and `pipeline_base_dir` are now global settings, defined in `config.yaml`. The previous CLI parameters have been removed.
```

Comment:
very tiny:
let's make the statement `The previous CLI parameters have been removed.` bold with a warning
https://squidfunk.github.io/mkdocs-material/reference/admonitions/#+type:warning
WDYT?

Reply:
good idea! I'll check to see if I did it correctly